configbuilder.go

```diff
@@ -0,0 +1,78 @@
+package mazzaroth
+
+import (
+	"crypto/ed25519"
+
+	"github.com/kochavalabs/mazzaroth-xdr/xdr"
+	"github.com/pkg/errors"
+)
+
+type ConfigBuilder struct {
+	sender                *xdr.ID
+	channel               *xdr.ID
+	nonce                 uint64
+	blockExpirationNumber uint64
+	ownerAddress          *xdr.ID
+	adminAddresses        []xdr.ID
+}
+
+func (cfgb *ConfigBuilder) Config(sender, channel *xdr.ID, nonce, blockExpirationNumber uint64) *ConfigBuilder {
+	cfgb.sender = sender
+	cfgb.channel = channel
+	cfgb.nonce = nonce
+	cfgb.blockExpirationNumber = blockExpirationNumber
+	return cfgb
+}
+
+func (cfgb *ConfigBuilder) Owner(address *xdr.ID) *ConfigBuilder {
+	cfgb.ownerAddress = address
+	return cfgb
+}
+
+func (cfgb *ConfigBuilder) Admins(addresses ...*xdr.ID) *ConfigBuilder {
```

Comment:
Could this be changed to not pass the pointer?  Seems odd to have to loop and dereference these here.  The Arguments list in callbuilder doesn't do this.

Reply:
> 

changed